src/codemagic/models/export_options.py

```diff
@@ -177,8 +177,8 @@ def set_value(self,
         else:
             setattr(self, field_name, value)
 
-    def update(self, **other_options):
-        for field_name, value in other_options:
+    def update(self, other_options):
```

Comment:
Add type hint to the `other_options` parameter. It is supposed to be a non-null `Dict` now at all times.